compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/SyntaxKind.java

```diff
@@ -26,109 +28,109 @@ public enum SyntaxKind {
-    BASE64_KEYWORD(240, "base64"),
-    MATCH_KEYWORD(241, "match"),
-    CONFLICT_KEYWORD(242, "conflict"),
-    LIMIT_KEYWORD(243, "limit"),
-    JOIN_KEYWORD(244, "join"),
-    OUTER_KEYWORD(245, "outer"),
-    EQUALS_KEYWORD(246, "equals"),
-    CLASS_KEYWORD(247, "class"),
-    ORDER_KEYWORD(248, "order"),
-    BY_KEYWORD(249, "by"),
-    ASCENDING_KEYWORD(250, "ascending"),
-    DESCENDING_KEYWORD(251, "descending"),
+    RETURNS_KEYWORD(200, LexerTerminals.RETURNS),
+    RETURN_KEYWORD(201, LexerTerminals.RETURN),
+    EXTERNAL_KEYWORD(202, LexerTerminals.EXTERNAL),
+    TRUE_KEYWORD(203, LexerTerminals.TRUE),
+    FALSE_KEYWORD(204, LexerTerminals.FALSE),
+    IF_KEYWORD(205, LexerTerminals.IF),
+    ELSE_KEYWORD(206, LexerTerminals.ELSE),
+    WHILE_KEYWORD(207, LexerTerminals.WHILE),
+    CHECK_KEYWORD(208, LexerTerminals.CHECK),
+    CHECKPANIC_KEYWORD(209, LexerTerminals.CHECKPANIC),
+    PANIC_KEYWORD(210, LexerTerminals.PANIC),
+    CONTINUE_KEYWORD(211, LexerTerminals.CONTINUE),
+    BREAK_KEYWORD(212, LexerTerminals.BREAK),
+    TYPEOF_KEYWORD(213, LexerTerminals.TYPEOF),
+    IS_KEYWORD(214, LexerTerminals.IS),
+    NULL_KEYWORD(215, LexerTerminals.NULL),
+    LOCK_KEYWORD(216, LexerTerminals.LOCK),
+    FORK_KEYWORD(217, LexerTerminals.FORK),
+    TRAP_KEYWORD(218, LexerTerminals.TRAP),
+    IN_KEYWORD(219, LexerTerminals.IN),
+    FOREACH_KEYWORD(220, LexerTerminals.FOREACH),
+    TABLE_KEYWORD(221, LexerTerminals.TABLE),
+    KEY_KEYWORD(222, LexerTerminals.KEY),
+    LET_KEYWORD(223, LexerTerminals.LET),
+    NEW_KEYWORD(224, LexerTerminals.NEW),
+    FROM_KEYWORD(225, LexerTerminals.FROM),
+    WHERE_KEYWORD(226, LexerTerminals.WHERE),
+    SELECT_KEYWORD(227, LexerTerminals.SELECT),
+    START_KEYWORD(228, LexerTerminals.START),
+    FLUSH_KEYWORD(229, LexerTerminals.FLUSH),
+    CONFIGURABLE_KEYWORD(230, LexerTerminals.CONFIGURABLE),
+    WAIT_KEYWORD(231, LexerTerminals.WAIT),
+    DO_KEYWORD(232, LexerTerminals.DO),
+    TRANSACTION_KEYWORD(233, LexerTerminals.TRANSACTION),
+    TRANSACTIONAL_KEYWORD(234, LexerTerminals.TRANSACTIONAL),
+    COMMIT_KEYWORD(235, LexerTerminals.COMMIT),
+    ROLLBACK_KEYWORD(236, LexerTerminals.ROLLBACK),
+    RETRY_KEYWORD(237, LexerTerminals.RETRY),
+    ENUM_KEYWORD(238, LexerTerminals.ENUM),
+    BASE16_KEYWORD(239, LexerTerminals.BASE16),
+    BASE64_KEYWORD(240, LexerTerminals.BASE64),
+    MATCH_KEYWORD(241, LexerTerminals.MATCH),
+    CONFLICT_KEYWORD(242, LexerTerminals.CONFLICT),
+    LIMIT_KEYWORD(243, LexerTerminals.LIMIT),
+    JOIN_KEYWORD(244, LexerTerminals.JOIN),
+    OUTER_KEYWORD(245, LexerTerminals.OUTER),
+    EQUALS_KEYWORD(246, LexerTerminals.EQUALS),
+    CLASS_KEYWORD(247, LexerTerminals.CLASS),
+    ORDER_KEYWORD(248, LexerTerminals.ORDER),
+    BY_KEYWORD(249, LexerTerminals.BY),
+    ASCENDING_KEYWORD(250, LexerTerminals.ASCENDING),
+    DESCENDING_KEYWORD(251, LexerTerminals.DESCENDING),
     UNDERSCORE_KEYWORD(252, "_"),
     NOT_IS_KEYWORD(253, "!is"),
```

Comment:
We need to handle `!is` as well

Reply:
Updated 6bbce7083c293a84ed7c20878a8efe55a7602504